main.py

```diff
@@ -28,69 +34,89 @@ def change_img_pixel_format():
 	the display Surface. Necessary for blitting images to the screen faster.
 	"""
 	background_module.bg = background_module.bg.convert()
+

```

Comment:
Clarification required: Is it really necessary to extract the alpha channel of this image? Why can't you do it outside the program and remove this conversion part?

Reply:
@Dcruise546  The convert() method converts the pixel format of the image(i.e pygame.Surface) to that of the display (i.e pygame.display). Doing this ourselves at the start saves computation to a large extent, else it would have to convert it in every game loop.
But this wipes out alpha channels if there are any in the image. So to preserve the alpha channels in the image during conversion convert_alpha() is used.
I really don't know how it works exactly and couldn't find any good resources for that. 
This discussion explains it a bit.
[https://gamedev.stackexchange.com/questions/81280/pygame-surfaces-which-and-when-do-i-need-to-convert#:~:text=convert()%20is%20used%20to,set_mode()%20.](URL)

As far as my knowledge is concerned, since it is done internally by Pygame it cannot be done outside the program. I will try to optimize this part of the code in the upcoming branches.